README.md

```diff
@@ -228,13 +240,43 @@ In this section, we'll add the necessary configuration to work with GitHub Actio
     }
     ```
 
-    Add a similar function for each example that you want to run in an integration test.  For examples written in other languages, create similar files for `examples_${LANGUAGE}_test.go`.
-1. Ensure that any required secrets (API keys, etc.) are present in GitHub.
+1. Add a similar function for each example that you want to run in an integration test.  For examples written in other languages, create similar files for `examples_${LANGUAGE}_test.go`.
+
+1. You can run these tests locally via Make:
+
+    ```bash
+    make test
+    ```
+
+    You can also run each test file separately via test tags:
+
+    ```bash
+    cd examples && go test -v -tags=nodejs
+    ```
+
+## Configuring CI with GitHub Actions
+
+In this section, we'll add the necessary configuration to work with GitHub Actions for Pulumi's standard CI/CD workflows for providers.
+
+1. Generate GitHub workflows per [the instructions in the ci-cmgmt repository](https://github.com/pulumi/ci-mgmt/#readme) and copy to `.github/` in this repository.
+
+1. Ensure that any required secrets are present as repository-level [secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) in GitHub.  These will be used by the integration tests during the CI/CD process.
 
 ## Final Steps
 
+1. Ensure all required configurations (API keys, etc.) are documented in README-PROVIDER.md.
+
 1. Replace this file with the README for the provider and push your changes:
 
     ```bash
     mv README-PROVIDER.md README.md
     ```
+
+1. *For Pulumi engineers only:* The final build step in your Action, `publish_SDKs` will fail the first time. 
```

Comment:
My (potentially uninformed) impression is that the entire instructions are geared toward Pulumi engineers, so this may be unnecessary.  Don't sweat this item, tho.

Reply:
Nit: Use the action title "Publish SDKs" instead of the `name` attribute, since the former is what displays on the GHA UI.

Reply:
> My (potentially uninformed) impression is that the entire instructions are geared toward Pulumi engineers, so this may be unnecessary. Don't sweat this item, tho.

Yeah I was waffling back and forth on this. I'll take int out :)